### ISSUE TITLE
npm package is out of date

### ISSUE THREAD

**Issue by m90** [faisalman/ua-parser-js#288]
While the tests tell me that the library should be able to detect the Waterfox browser correctly, running the very same UA string against the library in a real-world environment returns `Firefox`. Using `0.7.17` and running this:

```js
var UaParserJs = require("ua-parser-js")

const u = new UaParserJs('Mozilla/5.0 (X11; Linux x86_64; rv:55.0) Gecko/20100101 Firefox/55.2.2 Waterfox/55.2.2')
console.log(u.getBrowser())
```
will return `Firefox`, see: https://runkit.com/embed/uz1srl5b80th

**Comment by m90** [comment]
It looks like this is due to the fact that the version that will be pulled from npm is not up to date with the repository:

```
frederik@computer[repo](branch) $ cat node_modules/ua-parser-js/src/ua-parser.js | grep waterfox
frederik@computer[repo](branch) $ cat node_modules/ua-parser-js/src/ua-parser.js | grep LIBVERSION
    var LIBVERSION  = '0.7.17',
    UAParser.VERSION = LIBVERSION;
```

**Comment by faisalman** [comment]
Fixed in `0.7.18`
